src/azure-cli/azure/cli/command_modules/appservice/commands.py

```diff
@@ -89,7 +89,7 @@ def load_command_table(self, _):
         g.custom_command('start', 'start_webapp', validator=validate_app_or_slot_exists_in_rg)
         g.custom_command('restart', 'restart_webapp', validator=validate_app_or_slot_exists_in_rg)
         g.custom_command('browse', 'view_in_browser')
-        g.custom_command('list-runtimes', 'list_runtimes')
+        g.custom_command('list-runtimes', 'list_runtimes_hardcoded')
```

Comment:
@calvinsID (Nit) can you add a comment here stating that we can move back to the original method once the backend API is updated & deployed

Reply:
Added!